src/internal.c

```diff
@@ -15296,9 +15296,9 @@ static void DtlsUpdateWindowGTSeq(word32 diff, word32* window)
             if (i < idx)
```

Comment:
relative to line 15281:
I think the comment isn't useful. Diff is incremented by one because the window is slid relative to the next expected sequence number, that is the current sequence number is incremented by one.

Reply:
Changed comment.